examples/visualization/plot_study.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualizing High-dimensional Parameter Relationships\n",
+    "\n",
+    "This notebook demonstrates various visualizations of studies in Optuna.\n",
+    "The hyperparameters of a neural network trained to classify images are optimized and the resulting study is then visualized using these features.\n",
+    "\n",
+    "**Note:** If a parameter contains missing values, a trial with missing values is not plotted."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing the Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import fetch_openml\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "\n",
+    "mnist = fetch_openml(name='Fashion-MNIST', version=1)\n",
+    "classes = list(set(mnist.target))\n",
+    "\n",
+    "# For demonstrational purpose, only use a subset of the dataset.\n",
+    "n_samples = 4000\n",
+    "data = mnist.data[:n_samples]\n",
+    "target = mnist.target[:n_samples]\n",
+    "\n",
+    "x_train, x_test, y_train, y_test = train_test_split(data, target)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining the Objective Function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.neural_network import MLPClassifier\n",
+    "\n",
+    "def objective(trial):\n",
+    "    \n",
+    "    clf = MLPClassifier(\n",
+    "        hidden_layer_sizes=(\n",
+    "            trial.suggest_int('n_units_l0', 32, 64),\n",
+    "            trial.suggest_int('n_units_l1', 32, 64),\n",
+    "            trial.suggest_int('n_units_l2', 32, 64),\n",
+    "        ),\n",
```

Comment:
I think we can use a for-loop to suggest `n_units_l0`, `n_units_l1`, and `n_units_l2`, and it reduces the redundancy of code.

Reply:
I actually thought that this was more comprehensible but okay I agree we can use some DRY. Fixed!